unpackaged/post/config/package.xml

```diff
@@ -28,6 +28,8 @@
         <members>hed__Relationship__c.hed__HEDA_Relationship_Compact_Layout</members>
         <members>hed__Term_Grade__c.K12_Term_Grade_Compact_Layout</members>
         <members>hed__Term__c.hed__HEDA_Term_Compact_Layout</members>
+        <members>hed__Test__c.K12_Test_Compact_Layout</members>
+        <members>hed__Test_Score__c.K12_Test_Score_Compact_Layout</members>
```

Comment:
I can see the list views in the UI, but looks like it is not added here, am I missing anything?

Reply:
@skosgi1 , the list view is in the test.object file.

Reply:
thank you! :)